lib/yard-sorbet/handlers/abstract_handler.rb

```diff
@@ -0,0 +1,23 @@
+# typed: strict
+# frozen_string_literal: true
+
+# Apllies an +@abstract+ tag to +abstract!+/+interface!+ modules (if not alerady present).
+class YARDSorbet::Handlers::AbstractHandler < YARD::Handlers::Ruby::Base
```

Comment:
I don't love this class name (one might think it's the abstract class for `YARDSorbet` handlers), let me know if you have other ideas.

Reply:
We could do `AbstractClassHandler`.